### ISSUE TITLE
Is this even usable?

### ISSUE THREAD

**Issue by 0xblur** [Crumbs350/pybookmark#1]
If so, how?
I've read the instructions by It has so many caveats that makes it impossible to understand in less than a day. I just want to merge my `.html` bookmark files.

**Comment by Crumbs350** [comment]
yes it works, or it did when coded. It has what you call caveats because I was being explicit about what problem it is designed to solve. I mostly use the view/edit features after merging and it has been awhile since I've merged files.

The code you want to merge bookmark html files as a single json structured output is   
`python3 /some_path/pybookmark/scripts/bookmarks_merge.py to_import_path/ to_export_path/`

There are bunch of options but that is the simplest. I don't believe I have an html exporter built in. If you want the combined file as html you could import to an empty firefox/chrome profile and dump the imported json bookmarks file as html. Or custom write a json exporter to write html.

Testing the merge function now in a fresh venv environment I've noticed a couple of things.

1. lxml was not marked as required, and should be: `pip3 install lxml ` removes a number of errors
2. the code is failing on building the merged json file. This is likely due to a partial code refactor I started awhile ago but since I mostly use the viewer didn't notice the bug. Sorry I'll take a look at it.

**Comment by Crumbs350** [comment]
I pushed an update to the repository. Installing from pypi, with the updates, will not work because currently pypi is rejecting my valid API tokens to push package updates.
